client/my-sites/earn/memberships/index.jsx

```diff
@@ -336,12 +336,12 @@ class MembershipsSection extends Component {
 					<h1>{ this.props.translate( 'Confirmation' ) }</h1>
 					<p>
 						{ this.props.translate(
-							'Do you want to disconnect Recurring Payments from your Stripe account?'
+							'Do you want to disconnect Payments from your Stripe account?'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.